layers/+lang/c-c++/packages.el

```diff
@@ -174,3 +175,7 @@
     :post-init
     (dolist (mode '(c-mode c++-mode))
       (spacemacs/setup-helm-cscope mode))))
+
+(defun c-c++/post-init-counsel-gtags ()
+  (spacemacs/counsel-gtags-define-keys-for-mode 'c-mode)
+  (spacemacs/counsel-gtags-define-keys-for-mode 'c++-mode))
```

Comment:
@TheBB I'll add variable to `gtags` layer, so people can decide which mode that the want gtags enable.